TableSchemer/Scheme.swift

```diff
@@ -55,7 +56,7 @@ public protocol Scheme: class {
         - parameter   cell:          The UITableViewCell being created.
         - parameter   relativeIndex: The cell index from the start of the scheme being configured.
      */
-    func configureCell(cell: UITableViewCell, withRelativeIndex relativeIndex: Int)
+    func configureCell(_ cell: UITableViewCell, withRelativeIndex relativeIndex: Int)
```

Comment:
I believe the verb and noun should go on the outside for Swift 3. Looking at the revised `UITableView` class, the methods adopt a similar style:

`open func insertSections(_ sections: IndexSet, with animation: UITableViewRowAnimation)`